input/size.go

```diff
@@ -349,6 +347,20 @@ const (
 	//      - witness_script (accepted_htlc_script)
 	AcceptedHtlcPenaltyWitnessSize = 1 + 1 + 73 + 1 + 33 + 1 + AcceptedHtlcScriptSize
 
+	// AcceptedHtlcSuccessWitnessSize 322 bytes
+	//      - number_of_witness_elements: 1 byte
+	//      - nil_length: 1 byte
+	//      - sig_alice_length: 1 byte
+	//      - sig_alice: 73 bytes
+	//      - sig_bob_length: 1 byte
+	//      - sig_bob: 73 bytes
+	//      - preimage_length: 1 byte
+	//      - preimage: 32 bytes
+	//      - witness_script_length: 1 byte
+	//      - witness_script (accepted_htlc_script)
+	AcceptedHtlcSuccessWitnessSize = 1 + 1 + 1 + 73 + 1 + 73 + 1 + 32 + 1 +
```

Comment:
These don't seem to be used anywhere...or is this for when _everything_ starts going through the sweeper? (death of the nursery, etc, etc)

Reply:
Yep, as mentioned in the PR body these will be used once SIGHASHSINGLE second-layer txns are batch in the sweeper. We can't resign the inputs, but we will want accurate weight estimates for the aggregate transaction.